discovery_engine_core/core/src/engine.rs

```diff
@@ -1887,13 +1565,13 @@ pub(crate) mod tests {
         );
 
         // reset the stacks and states
-        #[cfg(feature = "storage")]
-        {
-            engine.storage = SqliteStorage::init_storage_system(None, None, &|_| None)
-                .await
-                .unwrap()
-                .0;
-        }
+        engine.storage = SqliteStorage::init_storage_system(None, None, &|_| None)
+            .await
+            .unwrap()
+            .0;
+        // This triggers the trusted sources stack to also fetch articles
+        engine.endpoint_config.trusted_sources = Arc::new(RwLock::new(vec!["example.com".into()]));
+        engine.endpoint_config.excluded_sources = Arc::new(RwLock::new(Vec::new()));
```

Comment:
:grey_question: why was this not needed before?

Reply:
it was done before as well, it just moved out of the `InitConfig`.

Reply:
ah I see, ok thanks